smriprep/interfaces/fastsurfer.py

```diff
@@ -0,0 +1,673 @@
+    seg_log
+        Name and location for the log-file for the segmentation (FastSurferCNN).
+        Default '$SUBJECTS_DIR/$sid/scripts/deep-seg.log'
+    clean_seg
+        Flag to clean up FastSurferCNN segmentation
+    run_viewagg_on
+        Define where the view aggregation should be run on.
+        By default, the program checks if you have enough memory to run
+        the view aggregation on the gpu. The total memory is considered for this decision.
+        If this fails, or you actively overwrote the check with setting
+        ``run_viewagg_on cpu``, view agg is run on the cpu.
+        Equivalently, if you define ``--run_viewagg_on gpu``, view agg will be run on the gpu
+        (no memory check will be done).
+    no_cuda
+        Flag to disable CUDA usage in FastSurferCNN (no GPU usage, inference on CPU)
+    batch
+        Batch size for inference. Default 16. Lower this to reduce memory requirement
+    order
+        Order of interpolation for mri_convert T1 before segmentation
+        ``(0=nearest, 1=linear(default), 2=quadratic, 3=cubic)``
+
+
+    Surface pipeline arguments
+    --------------------------
+    fstess
+        Use ``mri_tesselate`` instead of marching cube (default) for surface creation
+    fsqsphere
+        Use FreeSurfer default instead of
+        novel spectral spherical projection for qsphere
+    fsaparc
+        Use FS aparc segmentations in addition to DL prediction
+        (slower in this case and usually the mapped ones from the DL prediction are fine)
+    surfreg
+        Create Surface-Atlas ``sphere.reg`` registration with FreeSurfer
+        (for cross-subject correspondence or other mappings)
+    parallel
+        Run both hemispheres in parallel
+    threads
+        Set openMP and ITK threads
+
+
+    Other
+    ----
+    py
+        which python version to use.
+        Default ``python3.6``
+    seg_only
+        only run FastSurferCNN
+        (generate segmentation, do not run the surface pipeline)
+    surf_only
+        only run the surface pipeline ``recon_surf``.
+        The segmentation created by FastSurferCNN must already exist in this case.
+
+
+    """
+    sd = Directory(
+        exists=True,
+        argstr="--sd %s",
+        mandatory=False,
+        desc="Subjects directory"
+    )
+    sid = traits.String(
+        exists=True,
+        argstr="--sid %s",
+        mandatory=False,
+        desc="Subject ID"
+    )
+    t1 = File(
+        exists=True,
+        mandatory=False,
+        argstr="--t1 %s",
+        desc="T1 full head input (not bias corrected, global path)"
+    )
+    fs_license = File(
+        exists=True,
+        mandatory=False,
+        argstr="--fs_license %s",
+        desc="Path to FreeSurfer license key file."
+    )
+    seg = File(
+        exists=True,
+        mandatory=False,
+        argstr="--seg %s",
+        desc="Global path with filename of segmentation"
+    )
```

Comment:
Conventionally, we use `_file` to indicate that an input is a file. Also, all paths are assumed to be absolute, so no need to specify here.

```suggestion
    seg = File(
        exists=True,
        argstr="--seg %s",
        desc="Pre-computed segmentation file"
    )
```